src/app/app.component.html

```diff
@@ -1,3 +1,9 @@
-<router-outlet>
+<div class="spinner" *ngIf="isLoading">
```

Comment:
Are these ngIf checks necessary? Try to move the spinner implementation to `index.html` and place it inside `<app-root>` - it should work without any additional logic.